Misc/NEWS.d/next/Library/2019-12-02-21-03-41.bpo-36470.Gy9Sff.rst

```diff
@@ -0,0 +1,6 @@
+Allow dataclasses.replace() to work InitVar with default values
```

Comment:
Can you make this a single paragraph? Maybe something simpler like "Fix dataclasses with InitVars and replace(). Patch by Claudiu Popa."